gitpod-io/gitpod
# [Usage-based] Update billing page layout for teams and personal accounts

## Description


Update billing page layout for usage-based teams and personal accounts.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/13405

## How to test


1. Create a team called "Gitpod [Something]"
2. Check the Team Billing UI
3. Upgrade the team by adding a credit card
4. Check the Team Billing UI again
5. Go to `/billing` (your individual billing settings)
6. Check your User Billing UI
7. Upgrade your user by adding a credit card
8. Check your User Billing UI again

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
